### DISCUSSION TITLE
¿Tienen algún canal de Youtube que recomienden para informarnos sobre el Sector IT?

### DISCUSSION THREAD

**Discussion by Mar-rbw** [Launch-X-Latam/MisionFrontEnd#96] [General]
Buenos días, buenas tardes ¿Cómo están?
Soy su compañero Marcelo, un estudiante cualquiera.
Con otra interacción para mis compañeros de Launch X .
Siendo la pregunta de hoy: 

**¿Tienen algún canal de Youtube que recomienden para informarnos sobre el Sector IT?**

Saludos.

**Comment by Amhedriel** [comment]
Buenas,

Ya dieron sugerencias en otras discusiones, te recomiendo que los busques y los revises, así no nos repetimos, saludos.
